src/Go.CodeAnalysis/Parser/ImportNode.cs

```diff
@@ -6,29 +6,101 @@
         }
 
         public SnapshotSegment ImportExtent { get; }
 
+        public ImmutableArray<ImportDeclarationNode> ImportDeclarations { get; }
+
         public static bool TryParse(Lexer lexer, IList<Error> errors, out ImportNode parseNode)
         {
             var start = lexer.CurrentLexeme.Extent.Start;
+            parseNode = null;
+
+            // Parse "import" keyword.
+            if (!lexer.IsCorrectLexemeKeywordOrReportError(Keywords.Import, errors)
+                || !lexer.TryAdvanceLexemeOrReportError(errors))
+            {
+                errors.Add(new Error(lexer.CurrentLexeme.Extent, "Import keyword expected."));
+                return false;
+            }
+
+            // Single line import (string immediately following import keyword).
+            if (!lexer.IsCorrectLexemeOperatorOrReportError('(', errors))
+            {
+                ImportDeclarationNode singleLineDeclaration = null;
+                if (ImportDeclarationNode.TryParse(lexer, errors, out singleLineDeclaration))
+                {
+                    var singleLineImportExtent = new SnapshotSegment(lexer.Snapshot, start, lexer.CurrentLexeme.Extent.End - start);
+                    parseNode = new ImportNode(
+                        singleLineImportExtent,
+                        ImmutableArray.Create(singleLineDeclaration));
+                    return true;
+                }
+                else
+                {
+                    return false;
+                }
+            }
+
+            // Block import.
 
-            if (!lexer.IsCorrectLexemeKeywordOrReportError(Keywords.Import, errors) ||
-                !lexer.TryAdvanceLexemeOrReportError(errors) ||
-                !lexer.IsCorrectLexemeTypeOrReportError(LexemeType.String, errors))
+            // Advance over opening parenthesis.
+            if (!lexer.TryAdvanceLexemeOrReportError(errors))
             {
-                parseNode = null;
                 return false;
             }
 
-            var extent = new SnapshotSegment(lexer.Snapshot, start, lexer.CurrentLexeme.Extent.End - start);
-            parseNode = new ImportNode(extent, lexer.CurrentLexeme.Extent);
+            var declarationArrayBuilder = ImmutableArray.CreateBuilder<ImportDeclarationNode>();
+            bool isEndOfImportBlock = false;
+
+            // Parse each import declaration.
+            while(!isEndOfImportBlock)
+            {
+                // Parse declaration and advance over.
+                ImportDeclarationNode importLineNode = null;
+                if (ImportDeclarationNode.TryParse(lexer, errors, out importLineNode)
+                    && lexer.TryAdvanceLexemeOrReportError(errors))
+                {
+                    declarationArrayBuilder.Add(importLineNode);
+                }
+                else
+                {
+                    return false;
+                }
+
+                if (lexer.IsCorrectLexemeOperatorOrReportError(',', errors)
```

Comment:
I think this might need to be `IsCorrectLexemeOperator()` or we'll report errors.

Reply:
Added those methods without error reporting.